onnx_chainer/mapping.py

```diff
@@ -91,6 +91,7 @@
     'MaxPooling2D': (1, 8),
     'MaxPoolingND': (1, 8),
     'ROIPooling2D': (1,),
+    'Unpooling2D': (1, 9),
```

Comment:
`Upsample` is supported from opset=7, but actually ONNX chainer only support from opset=7, not cause an issue.

Reply:
Change the minimal version number to 7. As it does nothing useful, I've sent #112 for review.

https://github.com/chainer/onnx-chainer/pull/112